torchgeo/trainers/utils.py

```diff
@@ -81,15 +83,16 @@ def load_state_dict(model: Module, state_dict: "OrderedDict[str, Tensor]") -> Mo
         )
         del state_dict["conv1.weight"]
 
-    if num_classes != expected_num_classes:
+    if expected_num_classes is None:
+        warnings.warn("pretrained model does not include class weights")
```

Comment:
Is this worth warning about? Most (all?) of our pretrained models lack the last layer since it will need to be replaced anyway.

Reply:
That's fine